server/graphql/v2/object/Expense.ts

```diff
@@ -634,6 +634,15 @@ export const GraphQLExpense = new GraphQLObjectType<ExpenseModel, express.Reques
           }
         },
       },
+      draftKey: {
+        type: GraphQLString,
+        description: '[Host Admin only] Key to access the draft of this expense',
+        async resolve(expense, _, req) {
+          if (await ExpenseLib.canSeeDraftKey(req, expense)) {
+            return expense.data?.draftKey;
+          }
+        },
+      },
```

Comment:
Given how sensitive it is, we should add tests in `test/server/graphql/v2/query/ExpenseQuery.test.js` to guarantee that only host admins can see it